Content.Shared/_DV/CosmicCult/Components/CosmicCorruptableComponent.cs

```diff
@@ -0,0 +1,13 @@
+using Robust.Shared.Prototypes;
+
+namespace Content.Shared._DV.CosmicCult.Components;
+
+/// <summary>
+/// Indicates that an entity will be converted to the given prototype when corrupted by the Cosmic Cult
+/// </summary>
+[RegisterComponent]
+public sealed partial class CosmicCorruptableComponent : Component
```

Comment:
the correct word is "corruptible"
```suggestion
public sealed partial class CosmicCorruptibleComponent : Component
```